### PR TITLE
0.10.1-osx-signed, 0.10.1-osx-unsigned, 0.10.1-win

### PR DESCRIPTION
**Pull request by midnightmagic**
Sigs all seem to match. Still issue getting LXC build matching. OSX pieces built with KVM instead.

### CODE REVIEW

File: 0.10.1-win/midnightmagic/0x28DF2724F241D8EE/bitcoin-win-0.10-build.assert [hunk 1]
```diff
@@ -0,0 +1,797 @@
+--- !omap
```
Comment (laanwj) [inline 0.10.1-win/midnightmagic/0x28DF2724F241D8EE/bitcoin-win-0.10-build.assert]: There seems to be a extra directory level in between here

Comment (midnightmagic) [inline 0.10.1-win/midnightmagic/0x28DF2724F241D8EE/bitcoin-win-0.10-build.assert]: Indeed. I'll resubmit.

**Comment by midnightmagic** [comment]
Directory-level fail. Correcting and resubmitting.

**Comment by midnightmagic** [comment]
Hrm. Right, didn't mean to close it. I've submitted c92c6e1967a999be693a93569c2be9b0582ada0f as a corrective submit. If you'd prefer I squash them into a single commit that's fine too. The prior sigs IMO have small non-zero value, but. Meh. Up to you I guess.

**Pull request merged**